screens/ActionsCongrats.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { Text, View, Button, Box} from 'native-base';
+import { COLORS } from '../common/constants';
+import { useNavigation } from '@react-navigation/native';
+import LeafIcon from '../assets/images/leafIcon.svg'
+
+
+
+
+function ActionsCongrats({route}) {
+  const {totalPoints, totalUserPoints} = route.params
+  const navigation = useNavigation()
+
+  const goTo = (route) => {
+    navigation.navigate(route)
+  }
+  
+  return (
+    <View style={styles.container as any}>
```

Comment:
Why is this 'as any'?

Reply:
When I applied the styles without the any I got a typescript error.